src/service/credit_service.py

```
@@ -0,0 +1,332 @@
+                    free_credit=result.get("freeCredit", 0),
+                    paid_credit=result.get("paidCredit", 0),
+                    total_available=result.get("totalAvailable", 0),
+                    can_use=result.get("canUse", True),
+                    checked_cost=result.get("checkedCost"),
+                )
+            else:
+                logger.error(f"크레딧 잔액 조회 실패: {data.get('message')}")
+                raise Exception(data.get("message", "잔액 조회 실패"))
+
+        except httpx.HTTPError as e:
+            logger.error(f"HTTP 에러 (잔액 조회): {e}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)